pack.sh

```diff
@@ -199,6 +199,18 @@ elif [ $level = 3 ]; then
         fi
     fi
 
+    for part in "boot/split_img/boot.PARTITION-dtb" "boot/split_img/boot.PARTITION-second" "recovery/split_img/recovery.PARTITION-dtb" "recovery/split_img/recovery.PARTITION-second" "vendor_boot/split_img/vendor_boot.PARTITION-dtb" "vendor_boot/split_img/vendor_boot.PARTITION-second"; do
+        if [ -f level3/$part ]; then
+            cp level1/_aml_dtb.PARTITION level3/$part
+        fi
+    done
+
+    for part in "boot_a/split_img/boot_a.PARTITION-dtb" "boot_a/split_img/boot_a.PARTITION-second" "recovery_a/split_img/recovery_a.PARTITION-dtb" "recovery_a/split_img/recovery_a.PARTITION-second" "vendor_boot_a/split_img/vendor_boot_a.PARTITION-dtb" "vendor_boot_a/split_img/vendor_boot_a.PARTITION-second"; do
+        if [ -f level3/$part ]; then
+            cp level1/_aml_dtb.PARTITION level3/$part
+        fi
+    done
+
```

Comment:
This would copy dtb before its packed, means old one.

Reply:
But new dtb just generated above by dtbTool and placed to level1/_aml_dtb.PARTITION

Reply:
it might also need to be gzipped if size is an issue.  Code below.

Reply:
Moved it back, dont have device with big dtb that need compression to test.